finetune/util/huggingface_interface.py

```diff
@@ -206,6 +207,9 @@ def _encode(self, texts):
                     token_ends = []
                     token_starts = []
                     for start, end in encoded.offsets:
+                        if token_ends:
+                            start = max(token_ends[-1], start)
```

Comment:
Super bizarre.  Any idea why this occurs?

Reply:
This is really just a difference between how huggingface and we think about chars that become more than 1 token.
They repeat the start and end, but we assign it to one of the tokens and set the other to 0 length.